Project.toml

```diff
@@ -1,10 +1,9 @@
 name = "KrylovKit"
 uuid = "0b1a1467-8014-51b9-945f-bf0ae24f4b77"
 authors = ["Jutho Haegeman"]
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
With the new functionality of the other PR, I would probably go to 0.9. Unless you need this as quickly as possible (I guess the other one would take a few more changes and one more CI run) and want to already tag this right now. I can then also change to 0.9 after merging #105 .

Reply:
I think it would be convenient for me if it was 0.8.3, since the RecursiveVec thing broke things... In the end, it does not matter too much, I changed this here to test it out with MPSKit